kpops/components/base_components/kubernetes_app.py

```diff
@@ -86,9 +94,10 @@ class KubernetesApp(PipelineComponent):
     class Config(CamelCaseConfig, DescConfig):
         pass
 
+    # When removed, mypy throws some errors about missign arguments in
```

Comment:
I can't reproduce this locally. are you sure this is still the case?

Reply:
It strangely isn't. I removed it.